spec/views/pugin/components/_status.html.haml_spec.rb

```diff
@@ -6,13 +6,17 @@
   context 'while not in a dissolution' do
 
   	before :each do 
-  	  module FlagHelper
-  		def self.dissolution? 
-  			false
-  		end
-  		def self.register_to_vote? 
-  			false
-  		end
+  	  module Pugin
```

Comment:
try using the RSpec `allow` syntax here - it's less code and doesn't completely overwrite the module